src/IssueViz.Security/Taint/TaintList/ViewModels/TaintIssuesControlViewModel.cs

```diff
@@ -130,6 +134,8 @@ private void SetCommands(ILocationNavigator locationNavigator)
             NavigateCommand = new DelegateCommand(
                 parameter =>
                 {
+                    telemetryManager.TaintIssueOpened();
```

Comment:
We haven't implement light-bulb navigation yet, but presumably we'll want to include that in the telemetry too (?). For now, I'd just add a note in the taint lightbulb ticket so we don't forget.

Reply:
Yep, I was thinking about that too, of maybe putting it in a more centralized place (e.g. if we have something that calls the selection service, since that's what both actions will do in addition to navigating).